src/org/janelia/mipav/plugins/worm/untwisting/PlugInDialogVolumeRenderDualJanelia.java

```diff
@@ -1132,9 +1163,11 @@ public void enableNext(int mode) {
 		lutPanel.removeAll();
 		opacityPanel.removeAll();
 		clipPanel.removeAll();
+		accuratePanel.removeAll();
 		tabbedPane.addTab("LUT", null, lutPanel);
 		tabbedPane.addTab("Opacity", null, opacityPanel);
 		tabbedPane.addTab("Clip", null, clipPanel);
+		tabbedPane.addTab("AccurateMode", null, accuratePanel);
```

Comment:
I think we renamed this "Selection"

Reply:
it is "Selection" in my eclipse, don't know why its not updated.